app/konnect/updates.md

```diff
@@ -3,11 +3,117 @@ title: Konnect Cloud Updates
 no_version: true
 ---
 
+<!-- vale off -->
+
 The updates contained in this topic apply to {{site.konnect_saas}},
 an application that lets you manage configuration for multiple runtimes
 from a single, cloud-based control plane, and provides a catalog of all deployed
 services.
 
+## July 2022
+
+### date TBA
+
+**New environment for {{site.konnect_short_name}}**
+: {{site.konnect_short_name}} is now available at [cloud.konghq.com](https://cloud.konghq.com), which replaces the [konnect.konghq.com](https://konnect.konghq.com) environment. The environment at `konnect.konghq.com` will no longer receive any updates, and will be deprecated in the near future.
+
+: Existing organizations will be automatically upgraded to the new {{site.konnect_short_name}} environment over the next 4-6 weeks. We will be contacting your organization administrator with more details on the upgrade process.
+
+: You can continue using your `konnect.konghq.com` account and wait until we migrate it, or create a new account and [manually migrate configuration to the new account](/konnect/getting-started/import).
+
+: The minimum supported {{site.base_gateway}} version in this environment is 2.5.0.1.
```

Comment:
```suggestion
: The minimum supported {{site.base_gateway}} version for this environment is 2.5.0.1.
```

This may be wrong if I'm misunderstanding the environment paradigm